### PR TITLE
fix: prevent dodging timout on coubeh

### PR DESCRIPTION
**Pull request by smaiill**
Now, even if it fails to add the reactions, for example, if the user deletes the message before the bot finishes adding the reactions, they will still be muted.

### CODE REVIEW

**Comment by smaiill** [comment]
not really.

**Review by luca-montaigut — CHANGES_REQUESTED** [review]
this create a regression 

image

**Comment by smaiill** [comment]
It should work now

**Comment by luca-montaigut** [comment]
> It should work now

Did you try it ? 🧐

File: src/modules/quoiFeur/quoiFeur.helpers.ts [hunk 1]
```diff
@@ -35,11 +35,16 @@ export const reactOnEndWithQuoi = async (message: Message) => {
...
   if (Math.random() <= probability) {
-    await reactWithCoubeh(message);
-    await message.member?.timeout(
-      ONE_MINUTE * 5,
-      `${message.member.displayName} have the cramptés`,
-    );
+    try {
+      await reactWithCoubeh(message);
+    } catch (error) {
+      console.log(error)
```
Comment (neolectron) [inline src/modules/quoiFeur/quoiFeur.helpers.ts:41]: Could you check if the reason of the throw is the disappearance of the message ?
We don't want to silence every other errors possible.
For any other errors just re-throw them.

You might need to change the fact that you timeout in the `finally` for it to work like we want.

**Review by neolectron — CHANGES_REQUESTED** [review]
Nice addition, making the check more robust is a good call !

**Pull request closed without merging**